usda_nutrition/management/commands/import_usda.py

```diff
@@ -1,6 +1,11 @@
 import csv
 import os
 import sys
+if (sys.version_info < (3, 0)):
+    reload(sys)
+    sys.setdefaultencoding('utf-8')
```

Comment:
I was able to get in today and look at why I did it that way, and the problem is that it was breaking on:
for row in reader:
    ....

That leaves me no module to work with, really. I think it has to be done this way but I could be wrong. Also, you may notice I did specify the encoding on models.py in the manner you described. I did this to prevent breaking if someone decides to make a new migration. migrate works fine, but makemigrations chokes on the character encoding on special characters.